internal/web3ext/web3ext.go

```diff
@@ -796,6 +796,18 @@ web3._extend({
 			params: 2,
 			inputFormatter: [null, null]
 		}),
+		new web3._extend.Method({
+			name: 'call',
+			call: 'trace_call',
+			params: 3,
+			inputFormatter: [web3._extend.formatters.inputCallFormatter, null, null]
+		}),
+		new web3._extend.Method({
+			name: 'callMany',
+			call: 'trace_callMany',
+			params: 3,
+			inputFormatter: [null, null, null]
```

Comment:
`callMany` uses an array of CallArgs. Should this have its own formatter?  Rel https://github.com/etclabscore/core-geth/pull/345#discussion_r604330305

Reply:
Resolved with e0b8ae0